metaflow/plugins/kfp/kfp.py

```diff
@@ -503,8 +505,12 @@ def build_kfp_dag(node: DAGNode, passed_in_split_indexes=None):
             dsl.get_pipeline_conf().add_op_transformer(pipeline_transform)
             dsl.get_pipeline_conf().set_parallelism(self.max_parallelism)
             dsl.get_pipeline_conf().set_timeout(self.workflow_timeout)
-            if ARGO_DEFAULT_TTL is not None:  # # if None, we use the Argo defaults
-                dsl.get_pipeline_conf().set_ttl_seconds_after_finished(ARGO_DEFAULT_TTL)
+            if (
+                KFP_TTL_SECONDS_AFTER_FINISHED is not None
+            ):  # if None, we use the Argo defaults
```

Comment:
technically, KFP uses the Argo defaults

Reply:
Addressed in https://github.com/zillow/metaflow/pull/42